types/google-apps-script/google-apps-script.card.d.ts

```diff
@@ -0,0 +1,653 @@
+             *
+             * Using OnClose may cause OpenAs to be ignored; if the client platform cannot support both selected values together, OnClose takes precedence.
+             */
+            OpenAs: typeof OpenAs;
+            /**
+             * Type of selection input.
+             */
+            SelectionInputType: typeof SelectionInputType;
+            /**
+             * Creates a new `Action`.
+             */
+            newAction(): Action;
+            /**
+             * Creates a new `ActionResponseBuilder`.
+             */
+            newActionResponseBuilder(): ActionResponseBuilder;
+            /**
+             * Creates a new `AuthorizationAction`.
+             */
+            newAuthorizationAction(): AuthorizationAction;
+            /**
+             * Creates a new `AuthorizationException`.
+             */
+            newAuthorizationException(): AuthorizationException;
+            /**
+             * Creates a new `ButtonSet`.
+             */
+            newButtonSet(): ButtonSet;
+            /**
+             * Creates a new `CardAction`.
+             */
+            newCardAction(): CardAction;
+            /**
+             * Creates a new `CardBuilder`.
+             */
+            newCardBuilder(): CardBuilder;
+            /**
+             * Creates a new `CardHeader`.
+             */
+            newCardHeader(): CardHeader;
+            /**
+             * Creates a new `CardSection`.
+             */
+            newCardSection(): CardSection;
+            /**
+             * Creates a new `ComposeActionResponseBuilder`.
+             */
+            newComposeActionResponseBuilder(): ComposeActionResponseBuilder;
+            /**
+             * Creates a new `Image`.
+             */
+            newImage(): Image;
+            /**
+             * Creates a new `ImageButton`.
+             */
+            newImageButton(): ImageButton;
+            /**
+             * Creates a new `KeyValue`.
+             */
+            newKeyValue(): KeyValue;
+            /**
+             * Creates a new `Navigation`.
+             */
+            newNavigation(): Navigation;
+            /**
+             * Creates a new `Notification`.
+             */
+            newNotification(): Notification;
+            /**
+             * Creates a new `OpenLink`.
+             */
+            newOpenLink(): OpenLink;
+            /**
+             * Creates a new `SelectionInput`.
+             */
+            newSelectionInput(): SelectionInput;
+            /**
+             * Creates a new `SuggestionsResponseBuilder`.
+             */
+            newSuggestionsResponseBuilder(): SuggestionsResponseBuilder;
```

Comment:
definition of `newSuggestions()` seems lacking.

Reply:
Done 👍